IM_calculation/Advanced_IM/advanced_IM_factory.py

```diff
@@ -48,15 +51,14 @@ def compute_ims(accelerations, configuration, adv_im_out_dir):
             out_dir = os.path.join(adv_im_out_dir, im)
 
             im_config = config[im]
-            components = ["000", "090", "ver"]
             script = [
                 "python",
                 os.path.join(advanced_im_dir, im_config["script_location"]),
                 "--OpenSees_path",
                 configuration.OpenSees_path,
             ]
             script.extend(
-                [get_acc_filename(f_dir, station_name, x) for x in components]
```

Comment:
Ummmmm. Wanna have a closer look at what you've done?

Reply:
Dictionary iteration will always stay in the same order, until modified by insertion and such.
ref: [https://docs.python.org/2/library/stdtypes.html#dict.items](https://docs.python.org/2/library/stdtypes.html#dict.items)
re-order the dictionary should solve the 'changed order' issue.

Reply:
Can't reorder it either. That's the order that we get the components from BBSeis.

Reply:
since the order of this doesnt really affect the result or usage. i'm marking this as resolved for now.

Reply:
Giving that link a read. It says that 'Keys and values are listed in an arbitrary order which is non-random'. 

Aka, even though this time it does actually come out correctly, iterating of a dictionary keys doesn't guarantee that the order will be the same as we inserted into the dictionary.

Reply:
Actually, in python 3.7 it is guaranteed to preserve order. But I feel like changing this is just asking for bugs / trouble later. Aka, we don't lose anything being more explicit.